src/server/service/scenarios/endpoints.js

```diff
@@ -30,6 +28,7 @@ exports.addScenario = asyncMiddleware(async function addScenarioAsync(
 
     if (!userId || !title || !description) {
         const scenarioCreateError = new Error(
+            // eslint-disable-next-line quotes
```

Comment:
was this an error between eslint/prettier?  curious why it's double quoted in the first place.

Reply:
Yeah, they seemed to disagree and wanted me to use:

```
`"The scenario's title and description must be provided by a valid user"` 
```

Using single quotes tripped up prettier for some reason :/